clients/lightrider/lightrider-entry.js

```diff
@@ -52,22 +53,17 @@ async function runLighthouseInLR(connection, url, flags, lrOpts) {
 
   try {
     const runnerResult = await lighthouse(url, flags, config, connection);
-    if (!runnerResult) return;
+    if (!runnerResult) throw new Error('Lighthouse finished without a runnerResult');
 
-    // pre process the LHR for proto
-    if (flags.output === 'json' && typeof runnerResult.report === 'string') {
-      // When LR is called with |internal: {keep_raw_response: true, save_lighthouse_assets: true}|,
-      // this code will log artifacts to raw_response.artifacts.
-      if (logAssets) {
-        // @ts-ignore - Regenerate the report, but tack on the artifacts.
-        runnerResult.lhr.artifacts = runnerResult.artifacts;
-        runnerResult.report = JSON.stringify(runnerResult.lhr);
-      }
-
-      return preprocessor.processForProto(runnerResult.report);
+    // When LR is called with |internal: {keep_raw_response: true, save_lighthouse_assets: true}|,
+    // this code will log artifacts to raw_response.artifacts.
+    if (logAssets) {
+      // @ts-ignore - Regenerate the report, but tack on the artifacts.
```

Comment:
maybe regenerate was referring to what it used to be doing? what about just `// @ts-ignore - piggyback the artifacts on the LHR.`